### ISSUE TITLE
Alternative formats for f-logs

### ISSUE THREAD

**Issue by Billiam** [fohristiwhirl/fluorine#13]
Writing an f-log as json is hard, since it can't be appended to. 

Since bots don't know when the game has ended, writing it can't just be deferred until the last second.

A format like csv (😢 ) might be a better fit, since new lines could be added as the game plays out.

**Comment by fohristiwhirl** [comment]
Well. The number of turns is known so the bot can certainly add the final "]" at the right time.

**Comment by Billiam** [comment]
That's true if the game runs to the last frame, but won't be if the game ends for other reasons (like running out of ships, exception raised, etc).

**Comment by fohristiwhirl** [comment]
How about this - Fluorine will try and interpret incomplete f-logs:

* If adding a "]" to the end makes it valid, we will do so.
* If deleting a "," from the end and then adding a "]" will make it valid, we will do so.

Would that be satisfactory?

**Comment by Billiam** [comment]
Definitely!

**Comment by fohristiwhirl** [comment]
Give it a spin now.
